ReleaseChecklist.md

```diff
@@ -1,29 +1,39 @@
 ## Checklist for making a release:
 
 ### Requirements
- - [ ] Lauchpad (Ubuntu One) account
- - [ ] gnupg key (has to be version 1, gpg2 won't work) for `your-name@ethereum.org` created and uploaded
- - [ ] Readthedocs account, access to the Solidity project
- - [ ] Write access to https://github.com/ethereum/homebrew-ethereum
-
-### Documentation check
- - [ ] Run `make linkcheck` from within `docs/` and fix any broken links it finds. Ignore false positives caused by `href` anchors and dummy links not meant to work.
+ - [ ] Github account with access to [solidity](https://github.com/ethereum/solidity), [solc-js](https://github.com/ethereum/solc-js),
+       [solc-bin](https://github.com/ethereum/solc-bin), [homebrew-ethereum](https://github.com/ethereum/homebrew-ethereum),
+       [solidity-blog](https://github.com/ethereum/solidity-blog) and [solidity-portal](https://github.com/ethereum/solidity-portal) repositories.
+ - [ ] DockerHub account with push rights to the [``solc`` image](https://hub.docker.com/r/ethereum/solc).
```

Comment:
I don't have dockerhub account so not sure how that works in details. Do you need access to a specific image or do you just need to be a member of the [ethereum](https://hub.docker.com/u/ethereum) organization?

Reply:
I think being part of the ``ethereum`` organization is sufficient. But ultimately what you happens is that you'll push the ``solc`` image, so I'd say the current phrasing is fine.